instrumentation/opentelemetry-extension-annotations-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/extensionannotations/WithSpanSingletons.java

```diff
@@ -62,7 +62,9 @@ private static SpanKind spanKindFromMethodRequest(MethodRequest request) {
   }
 
   private static SpanKind spanKindFromMethod(Method method) {
-    WithSpan annotation = method.getDeclaredAnnotation(WithSpan.class);
+    application.io.opentelemetry.extension.annotations.WithSpan annotation =
```

Comment:
As far as I understand these will be removed from core repo in the next release so we would need to fix the version to the latest available. Would it make sense to set the version to the last non-deprecated version just to save the hassle of dealing with deprecated classes? Though, as you have already done all the work to make it build with deprecated classes it is fine this way as well.

Reply:
I believe the plan in the core repo is to keep publishing the deprecated artifact until the next major release (2.0)